tests/vmi_lifecycle_test.go

```diff
@@ -1428,7 +1428,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				nodeList := libnode.GetAllSchedulableNodes(kubevirt.Client())
 
 				if len(nodeList.Items) == 0 {
```

Comment:
Same here, it seems the check can be removed because now CI can filter these tests on unsupported envs

Reply:
Same as the other comment, I think this is still helpful